lib/routes/gelbooru/namespace.ts

```diff
@@ -0,0 +1,7 @@
+import type { Namespace } from '@/types';
+
+export const namespace: Namespace = {
+    name: 'Gelbooru | Free Anime and Hentai Gallery',
+    url: 'gelbooru.com',
+    description: `gelbooru posts`,
```

Comment:
```suggestion
    description: 'gelbooru posts',
```
Only use template literals when necessary.